src/tradingo/templates/instruments/yfinance.yaml

```
@@ -29,46 +36,46 @@
     start_date: "{{ data_interval_start }}"
     end_date: "{{ data_interval_end }}"
   depends_on:
-  {%for ticker in tickers %}
+    #  {%for ticker in tickers %}
     - "sample.{{ ticker }}"
-  {% endfor %}
+  #  {% endfor %}
   symbols_in:
     instruments: "instruments/{{ universe_name }}"
   load_args:
     no_date: true
   publish_args:
     astype: float
   symbols_out:
-  {% for occurence in ["open", "high", "low", "close", "volume"] %}
-  - prices/{{ universe_name }}.mid.{{ occurence }}
-  {% endfor %}
+    #  {% for occurence in ["open", "high", "low", "close", "volume"] %}
+    - prices/{{ universe_name }}.mid.{{ occurence }}
```

Is it an overkill to parametrise the `prices` library?

```suggestion
    - {{ out_prices_lib | default("prices") }}/{{ universe_name }}.mid.{{ occurence }}
```

(to be updated below too)